packages/core/src/browser/shell/side-panel-handler.ts

```diff
@@ -399,7 +425,7 @@ export class SidePanelHandler {
      *
      * @param menuId id of the menu to remove
      */
-    removeMenu(menuId: string): void {
+    removeBottomMenu(menuId: string): void {
```

Comment:
This is a breaking change for anyone calling `removeBottomMenu` and should be documented in the `changelog`.